packages/components/src/components/listing/QueriesListing.tsx

```diff
@@ -78,14 +81,18 @@ export class QueriesListing extends React.Component<QueriesListingProps, Queries
     }
 
     loadQueries(schemaName: string) {
-        fetchGetQueries(schemaName).then((queries) => {
-            this.setState(() => ({queries}));
-        });
+        fetchGetQueries(schemaName)
+            .then((queries) => {
+                this.setState(() => ({queries}));
+            })
+            .catch((error) => {
+                this.setState(() => ({error: error.exception}));
```

Comment:
Unless fetchGetQueries is already logging the error, it might be better to do a `console.error(error)` and then display a more friendly error message to the user (unless, of course, that exception is already, exceptionally, very friendly).

Reply:
I'll add a console.error here. But the current error message seems friendly to me, so I'll leave it being passed along:
"The schema name 'test' was not found within the folder '/Sample Manager'"